paper-input-container.html

```diff
@@ -260,18 +264,19 @@
         -webkit-appearance: none;
         text-align: inherit;
 
-        @apply(--paper-font-subhead);
```

Comment:
Like the other Polymer elements, if we can (because is cascading), set the external `apply()`s at the top. I turn back?